taskmgr/mock_test.go

```diff
@@ -0,0 +1,42 @@
+package taskmgr
+
+import (
+	"github.com/stretchr/testify/mock"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+	"github.com/taskcluster/taskcluster-client-go/tcclient"
+)
+
+type MockQueue struct {
+	mock.Mock
+}
+
+func (m *MockQueue) ReportCompleted(taskId, runId string) (*queue.TaskStatusResponse, *tcclient.CallSummary, error) {
```

Comment:
This is the exact call of the taskcluster-client-go queue client.  This is mocking those methods.